deploy/addons/gpu/kvm-nvidia-gpu.yaml.tmpl

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 The Kubernetes Authors All rights reserved.
```

Comment:
Thanks for pick up this task,
 I dont think we need to create a new Addon called KVM-nvidia...we should keep the addon intact, so we dont have to Maintain Two Copy of it.... I was hoping we have a solution that the Old Name still Works

what I had in mind was depreciation in the Flag Name, we already have an example of that

for example,

```
minikube start --help 
     --vm-driver='':
	DEPRECATED, use `driver` instead.
```

so we might have to refactor our Adddons package to allow an addon to have Two Names, or allowed it to have a depricated Name and when the user uses the depricated name we encourage them to use the new name.

and by the way this PR should probably Advice the users on the Docker Driver to use the --gpus flag instead

Reply:
@medyagh , thank you for your review, i misunderstood the need. i will try to refactor the work to:
- make both names working
- deprecate the old one with just notification 
for advising users on the docker driver, i double checked and it seems that on the documentation you already recommend that flag : https://minikube.sigs.k8s.io/docs/tutorials/nvidia/#docker